app/templates/relay/_hubs.html

```diff
@@ -1,17 +1,15 @@
 <table>
-  {% for h in hubs %}
-  {% if loop.first %}
   <tr>
     <th>Time</th>
-    <th>{% if h.hub_id is defined %}XBee{% endif %}</th>
-    <th>{% if h.pi_id is defined %}Pi ID{% endif %}</th>
+    <th>Hub ID</th>
+    <th>Pi ID</th>
```

Comment:
Good question, but don't worry this does the right thing. If a field was requested from the database, it will still be 'defined', even if it's null.

This is nice because multiple different pages use this partial template, but with different db queries requesting different fields. So before this change there was always an empty "XBee" column on the individual hub page (since we don't query the hub id since it's the same for every row) and there was always an empty Pi ID column on the hubs index page (since we don't query the π id because it is kinda useless).